cbv3_django_prototype/cbv3_django_prototype/tagging/admin.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib import admin
+from resources.models import TaggedItems
```

Comment:
I'm surprised to see a dependency on `resources.models` inside the `tagging` app. I would expect `resources` to depend on `tagging` and not the other way around. Interestingly this appears to be used on line 8 so I'm not sure I can just remove the dependency.

Reply:
This is case where Dango's file structure can be misleading.  While this file is parked inside the **`tagging`** app **directory**, its code is instructions to the **`admin`** app on how to display tags & tagging in the admin interface correctly.  I think these get placed with their apps so that there's no confusion on which piece of the admin they affect..but that creates confusion of a different type.

So while **`resources`** needs **`tagging`** to tag,  and **`tagging`** needs classes from **`taggit`** to override,  **`admin`** needs all three to be able to pull and display **`resources`** with their related **`tags`**:

![admin interface](https://user-images.githubusercontent.com/5923094/75628130-eee7bb00-5b8a-11ea-8841-f4d1e4d0e68e.png)